From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20Offenh=C3=A4user?= <offenhaeuser@protonmail.com>
Date: Tue, 7 Feb 2023 21:36:54 +0100
Subject: [PATCH] Remove use of strptime()

This is a hack to patch out strptime() from duktape, which is not being used by this project.
---
 src/thirdparty/duktape/duk_config.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/thirdparty/duktape/duk_config.h b/src/thirdparty/duktape/duk_config.h
index 4ac2e1d54dfc5da74136effbcb0e569c6ddea50a..b676781634c55ecb6c45f6cf01c0e9d1ee2afc75 100644
--- a/src/thirdparty/duktape/duk_config.h
+++ b/src/thirdparty/duktape/duk_config.h
@@ -221,7 +221,7 @@
 
 /* Generic Unix (includes Cygwin) */
 #if defined(__unix) || defined(__unix__) || defined(unix) || \
-    defined(DUK_F_LINUX) || defined(DUK_F_BSD)
+    defined(DUK_F_LINUX) || defined(DUK_F_BSD) || defined(__serenity__)
 #define DUK_F_UNIX
 #endif
 
@@ -840,7 +840,7 @@
 /* --- Generic UNIX --- */
 #define DUK_USE_DATE_NOW_GETTIMEOFDAY
 #define DUK_USE_DATE_TZO_GMTIME_R
-#define DUK_USE_DATE_PRS_STRPTIME
+#undef DUK_USE_DATE_PRS_STRPTIME
 #define DUK_USE_DATE_FMT_STRFTIME
 #include <time.h>
 #include <sys/time.h>
